.husky/pre-commit

```diff
@@ -2,7 +2,7 @@
 . "$(dirname "$0")/_/husky.sh"
 
 export NODE_ENV=dev
-npx lint-staged && npx tsc && npm test
+npx lint-staged && npx tsc
```

Comment:
👀

Reply:
I think run linting and tsc is a good compromise:
- they run fast
- they format code for you
- they can catch some bad stuff
- and most importantly, they don't fail when everything is ok

Tests are run by CI anyway. 

As mentioned before, I literally could not git rebase locally because of the tests failing ONLY when run by Husky, so I'd rather not have a tool be in my way while I'm trying to work - I can run tests manually and CI will run them anyway.

Reply:
i want husky and lint-staged gone altogether :p